library/src/main/java/com/schibsted/spain/barista/BaristaClickActions.java

```diff
@@ -15,17 +18,43 @@
 
 public class BaristaClickActions {
 
-  public static void click(@IdRes int id) {
+  private static final ResourceTypeChecker RESOURCE_TYPE_CHECKER = new ResourceTypeChecker();
```

Comment:
It's probably a good idea to just make the methods static. Having an instance doesn't add any value, and we're doing this ugly thing in two places already.
Not sure if that belongs to this PR though.